managers/docker-manager/DockerBindManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const eventmesh = require('../../data-access-layer/eventmesh');
+const logger = require('../../common/logger');
+const CONST = require('../../common/constants');
+const utils = require('../../common/utils');
+const BaseManager = require('../BaseManager');
+const DockerService = require('./DockerService');
+
+class DockerBindManager extends BaseManager {
+
+  init() {
+    const queryString = `state in (${CONST.APISERVER.RESOURCE_STATE.IN_QUEUE},${CONST.APISERVER.RESOURCE_STATE.DELETE})`;
+    return this.registerCrds(CONST.APISERVER.RESOURCE_GROUPS.BIND, CONST.APISERVER.RESOURCE_TYPES.DOCKER_BIND)
+      .then(() => this.registerWatcher(CONST.APISERVER.RESOURCE_GROUPS.BIND, CONST.APISERVER.RESOURCE_TYPES.DOCKER_BIND, queryString));
+  }
+
+  processRequest(requestObjectBody) {
+    return Promise.try(() => {
+        if (requestObjectBody.status.state === CONST.APISERVER.RESOURCE_STATE.IN_QUEUE) {
+          return this._processBind(requestObjectBody);
+        } else if (requestObjectBody.status.state === CONST.APISERVER.RESOURCE_STATE.DELETE) {
+          return this._processUnbind(requestObjectBody);
+        }
+      })
+      .catch(err => {
+        logger.error('Error occurred in processRequest', err);
```

Comment:
Have a reference like **DockerBindManager**. Follow in places.

Reply:
Done